com.obsproject.Studio.Plugin.Vaapi.yml

```diff
@@ -0,0 +1,30 @@
+id: com.obsproject.Studio.Plugin.Vaapi
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+seperate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Vaapi
+  libdir: lib/
```

Comment:
is this is wrong IMHO because it is not a relative path. The default should be be fine since `prefix` is set.